urllib3/connectionpool.py

```diff
@@ -22,7 +23,6 @@
     LocationValueError,
     MaxRetryError,
     ProxyError,
-    ConnectTimeoutError,
```

Comment:
this, and line 38, change the external API of this module.  Not sure whether that is OK.
The alternative would be to use `noqa`, or add `__all__`.